georocket-server/src/main/java/io/georocket/constants/ConfigConstants.java

```diff
@@ -13,6 +13,10 @@
   public static final String STORAGE_FILE_PATH = "georocket.storage.file.path";
   public static final String STORAGE_HDFS_DEFAULT_FS = "georocket.storage.hdfs.defaultFS";
   public static final String STORAGE_HDFS_PATH = "georocket.storage.hdfs.path";
+  public static final String STORAGE_INDEX_EMBEDDED = "georocket.storage.index.embedded";
```

Comment:
I propose `georocket.index.elasticsearch.embedded`. This allows us to later add `georocket.index.class` which would be similar to STORAGE_CLASS.